### PR TITLE
Added Weather plugin

### PR DESCRIPTION
**Pull request by SaitamaSama**

### CODE REVIEW

**Comment by SaitamaSama** [comment]
Some of the usages can be found out in this room: https://chat.stackoverflow.com/rooms/156726/jeevescandoit

**Comment by kelunik** [comment]
Please fix your email in your commits so it's properly linked to your GitHub account.

File: src/Plugins/Weather.php [hunk 1]
```diff
@@ -73,8 +74,9 @@ public function search(Command $command) {
 
     private function getFormattedWeather(\stdClass $data): string {
         return sprintf(
-            "*%s*. **Maximum Temperature:** %d C, **Minimum Temperature:** %d C. **Humidity:** %d%%",
+            "*%s*. **Current Temperature:** %s. **Maximum Temperature:** %s C, **Minimum Temperature:** %s C. **Humidity:** %d%%",
```
Comment (kelunik) [inline src/Plugins/Weather.php]: `°C`?

Comment (SaitamaSama) [reply src/Plugins/Weather.php]: Does seem more appropriate. Changed!

File: src/Plugins/Weather.php [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+     * @inheritdoc
+     */
+    public function getDescription(): string {
+        return "Returns the weather conditions for the specified city.";
+    }
+
+    public function getCommandEndpoints(): array {
+        return [new PluginCommandEndpoint('Search', [$this, 'search'], 'weather')];
+    }
+}
```
Comment (PeeHaa) [inline src/Plugins/Weather.php]: Newline

Comment (SaitamaSama) [reply src/Plugins/Weather.php]: Done

**Pull request closed without merging**